lms/static/js/dashboard/track_events.js

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
How does this JavaScript get included on the dashboard? I don't see it being included in any templates.